mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.cpp

```diff
@@ -24,12 +33,16 @@ MPFaceMeshDetector::MPFaceMeshDetector(int numFaces,
 
 absl::Status
 MPFaceMeshDetector::InitFaceMeshDetector(int numFaces,
+                                         /*cv::Mat cameraMatrix,*/
                                          bool with_attention,
                                          const char *face_detection_model_path,
                                          const char *face_landmark_model_path,
-                                         const char *face_landmark_with_attention_model_path) {
+                                         const char *face_landmark_with_attention_model_path/*,
+                                         const char *geometry_pipeline_metadata_landmarks_path*/,
+                                         int window_size_param,
+                                         float velocity_scale_param) {
```

Comment:
Please, combine these 2 parameters into a structure.

Reply:
Also, maybe we can also combine all of these parameters into the structure. It will make it easier to initialize the  MPFaceMeshDetector.